homeassistant/components/smtp/notify.py

```diff
@@ -136,23 +136,23 @@ def __init__(
         self.debug = debug
         self._verify_ssl = verify_ssl
         self.tries = 2
+        self._ssl_context = create_client_context() if self._verify_ssl else None
```

Comment:
Please avoid side effects in `__init__`

Instead create this in `get_service` and pass it in